docs/reference/aggregations/bucket/autodatehistogram-aggregation.asciidoc

```diff
@@ -258,6 +258,28 @@ Like with the normal <<search-aggregations-bucket-datehistogram-aggregation, `da
 scripts and value level scripts are supported. This aggregation does not however, support the `min_doc_count`, 
 `extended_bounds` and `order` parameters.  
 
+==== Minimum Interval parameter
+
+The `minimum_interval` allows the caller to specify the minimum rounding interval that should be used.
```

Comment:
Could you add a bit more detail, for example, to explain that the aggregation will start from the specified interval and that it will make the collection more efficient (faster) since it won't waste time collected at lower intervals.

Also we should list the available values for this parameter.

Reply:
addressed in 388127ca081535148689a05ed20ce456c398fa42